docs/Releases.md

```diff
@@ -13,6 +13,41 @@ use a date based versioning system.  Now, a release version can look
 like `17.11` where the "major" number is the year and the "minor" number
 is the month.
 
+## v19.07 (7/19): NFD library and example NFs, minor improvements and bug fixes.
+A CloudLab template is available with the latest release here: https://www.cloudlab.us/p/GWCloudLab/onvm
+
+**Performance**: This release includes a new macro `ENABLE_FLOW_LOOKUP` which will disable an expensive flow lookup for every incoming packet(instead packets are forwarded to the default service ID). This improves performance but flow lookup is still enabled by default for backward compatibility with other applications that use ONVM.
```

Comment:
```suggestion
**Performance**: This release includes a new macro `ENABLE_FLOW_LOOKUP` which controls whether a flow lookup is performed for every incoming packet. If disabled, all packets are forwarded to the default service ID which improves performance. The flow lookup is still enabled by default for backward compatibility with other applications that use ONVM.
```

Reply:
If disabled, does it forward to the default service ID or the default service *chain*?